README.md

```diff
@@ -133,17 +141,15 @@ Then do the following BIOS settings:
 - Disable wake on LAN/thunderbolt
 - Set boot mode to UEFI only, disable CSM support
 
-Now you can install macOS on your APFS or HFS+ formatted drive.
+Now you can boot your USB stick and install macOS on your APFS or HFS+ formatted drive. If it fails to boot, double check again the settings in the BIOS.
```

Comment:
The last part sounds a bit funny, maybe change to "double check your BIOS settings"

Reply:
Yep :)